test-engineering/contract-tests/client/README.md

```diff
@@ -1,7 +1,77 @@
 # client
 
-This directory contains a Python-based test framework for the contract tests.
+This directory contains a Python-based test framework for the contract tests. 
+The HTTP client used in the framework supports:
 
-The HTTP client used in the framework requests suggestions from Merino and
-performs checks against the responses. The framework implements response models
-for the Merino API.
+* Requests to Kinto (Remote Settings) for record population.
+* Requests for suggestions from Merino, with response checks.
+
+For more details on contract test design, refer to the contract-tests 
+[README][contract_tests_readme].
+
+## Scenarios
+
+The client is instructed on request and response check actions via scenarios recorded 
+in a file. A scenario is defined by a name, a description and steps.
+
+### Steps
+
+#### Kinto Service
+
+* The Kinto service scenario step populates the records used by the Merino service
+* A Kinto `request` requires 3 fields:
+  * `service` - Set the value to `kinto`, to direct requests to the Kinto service. 
+  * `filename` - Set the file with records to upload. The files are located in 
+                 '..\contract-tests\volumes\kinto'.
+  * `data_type` - Set to `data` or `offline-expansion-data`.
+* A Kinto service scenario step will not check a `response` defined in the scenario, 
+  but will raise an error in the event of an unexpected HTTP response.
+
+Example:
+```yaml
+- request:
+    service: kinto
+    filename: "data-01.json"
+    data_type: "data"
+```
+
+#### Merino Service
+
+* To direct requests to the Merino service, set the `service` value of `request` to 
+`merino`.
```

Comment:
Could you also describe the fields of this request like what you did for the Kinto service? A brief description (no need to cover every field of `content`) for the `response` field would be nice as well.

Reply:
change applied: https://github.com/mozilla-services/merino/pull/409/commits/670814d39d81946a96d7d0e48d99c0e89adce4fd

@ncloudioj I've added what I understand for the merino scenario steps, but may add more as I get to know the project better. Let me know what you think.

Reply:
Looks good, thanks!